web/app/models.py

```diff
@@ -170,12 +170,14 @@ def to_dict(self):
 
 
 RULESETS = ["AGA", "INTERNATIONAL"]
-PAIRINGTYPES = ["McMahon"]
-KOMI_VALUES = ['0', '0.5', '1', '1.5', '2', '2.5', '3.0', '3.5', '4', '4.5',
-    '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']
+PAIRINGTYPES = ["McMahon", "Temporary_Alternative"]
+KOMI_VALUES = ['0', '0.5', '5.5', '6.5', '7.5']
 TIE_BREAKS = ['SOS', 'SODOS', 'Face to Face Result', 'Random Procedure']
+MATCH_RESULTS = ['White Wins', 'Black Wins', 'Jigo', 'No Result',
+    'White Forfeit', 'Black Forfeit', 'Double Forfeit']
 
 class Tournament(db.Model):
+    __tablename__ = 'tournament'
```

Comment:
No, I don't think necessary. I started playing with tablenames for the Round table since 'round' exists at a build in Py function and I wanted to avoid collision and/or confusion. I continued playing with it when I made a mess out of dependencies within the datebase and wanted to feel more confident in the handles I was using on my tables. I _think_ the only tablename i'm actively using right now is 'tourny_round' for the Round table.